recipes/aiostream/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "aiostream" %}
+{% set version = "0.4.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiostream-{{ version }}.tar.gz
+  sha256: 36345ef1d21bedffe0878210a9123234ca8d29d8f33f0f073bea1271735f63cc
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - async-exit-stack
+    - python >=3.6
+
+test:
+  imports:
+    - aiostream
+    - aiostream.stream
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/vxgmichel/aiostream
+  summary: Generator-based operators for asynchronous iteration
+  license: GPL-2.0-or-later
```

Looks like it's actually
```suggestion
  license: GPL-3.0-or-later
```

---

Thanks! Don't know how I missed that earlier